app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/AnalyticsServiceCEImpl.java

```diff
@@ -164,7 +164,8 @@ public <T extends BaseDomain> Mono<T> sendObjectEvent(AnalyticsEvents event, T o
             return Mono.just(object);
         }
 
-        final String eventTag = event.getEventName() + "_" + object.getClass().getSimpleName().toUpperCase();
+        // In case of action execution, event.getEventName() only is used to support backward compatibility of event name
+        final String eventTag = event.equals(AnalyticsEvents.EXECUTE_ACTION) ? event.getEventName() : event.getEventName() + "_" + object.getClass().getSimpleName().toUpperCase();
```

Comment:
```suggestion
        final String eventTag = AnalyticsEvents.EXECUTE_ACTION.equals(event) ? event.getEventName() : event.getEventName() + "_" + object.getClass().getSimpleName().toUpperCase();
```
Nit: In this particular case, it doesn't matter. But as a practice, always invoke the `equals` function on the static variable. Avoids NPEs.

Reply:
Done